src/fe/inf_fe_map.C

```diff
@@ -187,6 +187,13 @@ Point InfFE<Dim,T_radial,T_map>::inverse_map (const Elem * inf_elem,
            +o(2)*p1(0)*p0(1)+o(0)*p0(2)*p1(1));
 
 
+        if(! std::isfinite(c_factor)){
```

Comment:
`std::isfinite` is C++11-only code, unfortunately.  In C99 there is an `isinf(x)` macro which might be possible to use instead.  I think we already require C99 for other things...